wordpress-mobile/WordPress-iOS
# Decoupling Authentication Flow: Mark VIII

### Details:
This PR splits the **LoginEpilogue** from the Login flow:

- New **LoginEpilogue.storyboard**
- New **WordPressAuthenticatorDelegate** API, to present the Epilogue
- Removed all of the LoginEpilogue references from **Signup.storyboard** / **EmailMagicLink.storyboard** / **Signup.storyboard**

cc @ScoutHarris @nheagy 
Thanks in advance!!

### To test:
In order to verify this PR, please, run a smoke test over the login flows, and make sure the Epilogue shows up at the very last step:

1. Log into a Self Hosted Site
2. Log into a Dotcom account, with the following variations:
   1.  No 2FA / No Social Auth
   2.  No 2FA / Social Login
   3.  2FA Protected / No Social Auth
   4.  2FA Protected / Social Auth
